ui/index.html

```diff
@@ -1,12 +1,28 @@
 <!DOCTYPE html>
 <html>
+
 <head>
     <title>Hello World </title>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+    <link rel="stylesheet" href="style.css">

```

Comment:
Do you think it'd be better to place the css file in a `css/` folder and rename it to something like `search.css`. I think it makes it easier to navigate.